nodejs/eks/examples/all_test.go

```diff
@@ -236,10 +236,6 @@ func Test_AllTests(t *testing.T) {
 	}
 
 	longTests := []integration.ProgramTestOptions{}
-	// longTests := shortTests
-	// for _, longTest := range longTests {
-	// 	longTest.RunUpdateTest = true
-	// }
```

Comment:
If we really don't need this - can we delete it?  If so - can we also clean up the rest of the code around here since `longTests` no longer is needed?